### DISCUSSION TITLE
Language Server Rewrite

### DISCUSSION THREAD

**Discussion by TwitchBronBron** [rokucommunity/brighterscript#1322] [General]
I've started this discussion to have a place to talk through issues with the rewrite of the language server (tracked in #993).

Steps to install:
1. Go to #993  in a web browser
2. scroll to the bottom and copy the latest release link from the rokucommunity-bot post. 
3. paste into `.vscode/settings.json` in the `brightscript.bsdk` key (create it if missing). 

Here's a quick example of how to install.

https://github.com/user-attachments/assets/8b24e812-5478-4fc1-8590-178fad106aee

**Comment by Null-Sound** [comment]
Hey, I'd love to get involved. How can I start helping around here?

**Reply by TwitchBronBron** [reply]
Hey! We'd love the help! Just to clarify, this specific discussion item (the language server rewrite) has already been completed and released. If you're using the latest version of the BrightScript Language extension for VSCode, you're already using the latest language server. 

If you're looking to start off easy, check out one of the `good first issue` items that sounds interesting: 
https://github.com/rokucommunity/brighterscript/issues?q=is%3Aissue%20state%3Aopen%20label%3A%22good%20first%20issue%22

Also, many of us hang out in the [RokuCommunity slack](https://join.slack.com/t/rokudevelopers/shared_invite/zt-4vw7rg6v-NH46oY7hTktpRIBM_zGvwA). Feel free to DM me or post in the #brighterscript channel.

Was there something specific you were interested in helping out with?

**Comment by TwitchBronBron** [comment]
This initiative has been completed. Closing this discussion
